basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,332 @@
+  expect(helpers.subtractFive("cat")).toBe(0);
+});
+
+it("should convert a string to a number if possible", () => {
+  expect(helpers.subtractFive("10")).toBe(5);
+  expect(helpers.subtractFive("10.5")).toBe(5.5);
+});
+
+//areSameLength
+
+it("should check if two strings have the same length", () => {
+  expect(helpers.areSameLength("cat", "dog")).toBe(true);
+});
+
+it("should convert one or both arguments to a string if passed a number or array", () => {
+  expect(helpers.areSameLength(222, 444)).toBe(true);
+  expect(helpers.areSameLength(222, "cat")).toBe(true);
+  expect(helpers.areSameLength("dog", 444)).toBe(true);
+  expect(helpers.areSameLength("dog", ["dog"])).toBe(true);
+  expect(helpers.areSameLength(["dog"], 444)).toBe(true);
+});
+
+it("should return false if either one or both arguments are not a number, string, or array", () => {
+  expect(helpers.areSameLength(undefined, "cat")).toBe(false);
+  expect(helpers.areSameLength(null, 33)).toBe(false);
+  expect(helpers.areSameLength("dog", null)).toBe(false);
+  expect(helpers.areSameLength(33, undefined)).toBe(false);
+});
+
+//areEqual
+
+it("should check if two numbers are equal", () => {
+  expect(helpers.areEqual(4, 4)).toBe(true);
+  expect(helpers.areEqual(4, 5)).toBe(false);
+});
+
+it("should convert strings to numbers if possible", () => {
+  expect(helpers.areEqual("10", "10")).toBe(true);
+  expect(helpers.areEqual("10.5", "10.5")).toBe(true);
+  expect(helpers.areEqual("10", 10)).toBe(true);
+  expect(helpers.areEqual(10, "10")).toBe(true);
+  expect(helpers.areEqual("10", "11")).toBe(false);
+});
+
+//lessThanNinety
+
+it("return true if the argument is a number less than 90", () => {
+  expect(helpers.lessThanNinety(89)).toBe(true);
+  expect(helpers.lessThanNinety(91)).toBe(false);
+});
+
+it("should convert strings to numbers if possible", () => {
+  expect(helpers.lessThanNinety("89")).toBe(true);
+  expect(helpers.lessThanNinety("89.9999999")).toBe(true);
+});
+
+//greaterThanFifty
+
+it("return true if the argument is a number greater than 50", () => {
+  expect(helpers.greaterThanFifty(51)).toBe(true);
+});
+
+it("should convert strings to numbers if possible", () => {
+  expect(helpers.greaterThanFifty("89")).toBe(true);
+  expect(helpers.greaterThanFifty("89.9999999")).toBe(true);
+});
+
+//add
+
+it("return the sum of two numbers added together", () => {
+  expect(helpers.add(89, 1)).toBe(90);
+});
+
+it("should convert strings to numbers if possible", () => {
+  expect(helpers.add("89", 1)).toBe(90);
+  expect(helpers.add(89, "1")).toBe(90);
+});
+
+//subtract
+
+it("return the results of one number subtracted from another", () => {
+  expect(helpers.subtract(91, 1)).toBe(90);
+});
+
+it("should convert strings to numbers if possible", () => {
+  expect(helpers.subtract("91", 1)).toBe(90);
+  expect(helpers.subtract(91, "1")).toBe(90);
+});
+
+//divide
+
+it("return the remainder of one number divided by another", () => {
```

Comment:
I think getRemainder is a separate function. This one should divide one number by another.